### PR TITLE
index.md: Add pacman for Arch Linux and derivatives

### PR DESCRIPTION
**Pull request by evanburkey**

### CODE REVIEW

**Comment by sudoankit** [comment]
Thank you @evanburkey. Can you please rebase the PR ( because of the typo ) and add a gif and submit ?

File: index.md [hunk 1]
```diff
@@ -100,7 +100,25 @@ DNF or Dandified Yum is the next generation version of the Yellowdog Updater, Mo
 - `dnf remove package_name` : Kicks out the specified package and flushes out all its files.
 
 > Before running the above commands, you have to either get sudo access by entering `su` and then your password or by preceding the command with `sudo`.
- 
+
+### `pacman` : Package manager. For those who like a little danger with their updates.
+
+Pacman is the package manager for Arch Linux and its derivatives. Some users joke that it rolls a d20 and blows your system up on a 1.
+
```
Comment (sudoankit) [inline index.md]: Great line!

File: index.md [hunk 2]
```diff
@@ -100,7 +100,25 @@ DNF or Dandified Yum is the next generation version of the Yellowdog Updater, Mo
...
+- `pacman -Syu` : Installs any updates available to installed packages on your system.
+
+- `pacman -S package_name` : Searches the Arch repository for the package, downloads and installs it on the system.
+
+- `pacman -R package_name` : Removes the specified package.
+
+- `pacman -Rd package_name` : Removes the specified package and its dependencies, but only if those dependencies are not required by any other package.
+
+![pacman.gif](geekygifs/pacman.gif)
+
```
Comment (sudoankit) [inline index.md]: The gif should have an example of `pacman -Rd package_name`?

Comment (evanburkey) [inline index.md]: Ack, can't believe I missed that. I'll get this ugly PR cleaned up in the next day or two

Comment (sudoankit) [inline index.md]: Alright. Waiting for it.

**Comment by evanburkey** [comment]
Ok, all cleaned up and ready to be reviewed.

**Comment by sudoankit** [comment]
Alright, I'll merge it though your commit message shouldn't be in past tense. 

`Added pacman for Arch Linux and derivatives`

A better one would be, 
`index.md: Add pacman for Arch Linux and derivatives`

If you want to reword the commit go ahead and experiment , if not it's fine I'll merge it tomorrow. :) 

Although I suggest you to follow [commit message rules](http://chris.beams.io/posts/git-commit/) (that link is a good example) next time you contribute.

**Comment by evanburkey** [comment]
Thanks for the advice and input, I'm still very new to Github so I appreciate the hand-holding while I stumble my way through this :)

**Comment by sudoankit** [comment]
Not the heading of this PR. 
I meant the commit message.

**Comment by evanburkey** [comment]
Commit edited.

**Comment by sudoankit** [comment]
Thanks a lot. I am closing and merging it.

**Pull request merged**